command/rootchain/emit/params.go

```diff
@@ -0,0 +1,37 @@
+package emit
+
+import "errors"
+
+const (
+	contractFlag = "contract"
+	walletsFlag  = "wallets"
+	amountsFlag  = "amounts"
+)
+
+var (
+	errWalletsMissing       = errors.New("wallet flag value is not provided")
+	errAmountsMissing       = errors.New("amount flag value is not provided")
+	errInconsistentAccounts = errors.New("wallets and amounts must be provided in same numbers")
```

Comment:
What do you think about changing this error message to:
`wallets and amounts must be provided in pairs`?

Reply:
Fixed [969c204](https://github.com/0xPolygon/polygon-edge/pull/773/commits/969c204c90702980f068399b6d9f3d5d3940fdfb)